README.md

```diff
@@ -16,171 +17,151 @@
-| Android secure storage<br/> library | SecureStorage is used for keeping private information in a safe mode without requiring a password or a fingerprint <br/>* Java<br/>* Kotlin | [repo](https://github.com/epam/Android-Secure-Storage) | |
+# Documentation
 
-# Themis Server
+[Documentation for Themis](https://docs.cossacklabs.com/themis/) contains the ever-evolving official docs, which covers everything from deployment guidelines to use cases, with brief explanations of cryptosystems and architecture behind the main Themis library. 
 
-If you'd like to experiment with Themis in a more interactive environment, check out [Themis Server](https://themis.cossacklabs.com) interactive debugging environment for Themis. Themis Server can verify and decrypt the code encrypted by Secure Cell or Secure Message and provides a fully-functional backend for Secure Session. Examples for many languages are available in [docs/examples/Themis-server](https://github.com/cossacklabs/themis/tree/master/docs/examples/Themis-server).
+Refer to the documentation to learn more about:
+- [cryptography in Themis](https://docs.cossacklabs.com/themis/crypto-theory/) ([Themis cryptosystems and attacks on them](https://docs.cossacklabs.com/themis/crypto-theory/cryptosystems/), [cryptography donors](https://docs.cossacklabs.com/themis/crypto-theory/cryptography-donors/), [key management advice](https://docs.cossacklabs.com/themis/crypto-theory/key-management/), [FIPS 140-2 and GOST](https://docs.cossacklabs.com/themis/crypto-theory/fips-and-gost/)),
+- [debugging and troubleshooting Themis](https://docs.cossacklabs.com/themis/debugging/) ([command-line utilities](https://docs.cossacklabs.com/themis/debugging/cli-utilities/), [Themis Server interactive web simulator](https://docs.cossacklabs.com/themis/debugging/themis-server/), [thread safety](https://docs.cossacklabs.com/themis/debugging/thread-safety/), [migration guidelines between Themis versions](https://docs.cossacklabs.com/themis/debugging/migration-guides/)) ,  
+- [Themis architecture](https://docs.cossacklabs.com/themis/architecture/),
+- [Themis security](https://docs.cossacklabs.com/themis/security/) ([Themis and OWASP](https://docs.cossacklabs.com/themis/security/owasp/), [Themis development security practices](https://docs.cossacklabs.com/themis/security/dev-security-practices/)),
+- [community behind Themis](https://docs.cossacklabs.com/themis/community/) ([contributing guidelines](https://docs.cossacklabs.com/themis/community/contributing/), [credits and honorable mentions](https://docs.cossacklabs.com/themis/community/credits/),  [projects that use Themis](https://docs.cossacklabs.com/themis/community/projects-that-use-themis/), [tutorials](https://docs.cossacklabs.com/themis/community/tutorials-workshops-talks/), [workshops and talks](https://docs.cossacklabs.com/themis/community/tutorials-workshops-talks/)).
```

Comment:
This hyperlinked fiesta is probably my favorite part of this PR 🤩

Reply:
for me it looks more-less fine in readme (questionable, but okay)

Reply:
Well, it's all blue, but we're not in Wikipedia. It should be readable.